.storybook/preview.ts

```diff
@@ -138,6 +138,7 @@ const preview: Preview = {
   decorators,
   globalTypes,
   loaders,
+  tags: ["autodocs"],
```

Comment:
**suggestion**: as auto-generated documentation is enabled for all stories by default, what do you think about disabling it for the Welcome Page stories?:

```tsx
// in .storybook/welcome-page/welcome.stories.js
export default {
  title: "Welcome",
  component: Welcome,
  parameters: {
    chromatic: {
      disableSnapshot: true,
    },
    layout: "fullscreen",
  },
  tags: ["!autodocs"], // 👈 disable autogen of docs page
};
```

This would prevent visitors landing on the generated docs page when they visit carbon.sage.com:

![Screenshot 2024-09-27 at 17 05 14](https://github.com/user-attachments/assets/c4a714a0-c7e7-4068-8efe-cfe091ec8c8f)

Reply:
Sure, good suggestion. I'll do that